### PR TITLE
develop: log: document the runtime logging

### PR DESCRIPTION
**Pull request by mojaves**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:
## 
## 
## 

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

Add short introduction, with reference to the full docs,
about how to configure the Vdsm logging at runtime.

Signed-off-by: Francesco Romani fromani@redhat.com

### CODE REVIEW

File: source/develop/developer-guide/vdsm/log-files.html.md [hunk 1]
```diff
@@ -36,6 +36,16 @@ Since oVirt 3.5.0 (VDSM 4.16.0), VDSM does not enable the overly verbose libvirt
 
 ## Reading the log
 
+### Verbosity and debug information
+
+Vdsm produces a huge amount of logging, and to improve the readability and to ease the troubleshooting, effort are ongoing to make the output tunable, so the logs regard one or few interesting few, while informations about other
```
Comment (pkliczewski) [inline source/develop/developer-guide/vdsm/log-files.html.md]: "so the logs regard one or few interesting few" no sure what it means

Comment (mojaves) [inline source/develop/developer-guide/vdsm/log-files.html.md]: Right, need to update

File: source/develop/developer-guide/vdsm/log-files.html.md [hunk 2]
```diff
@@ -36,6 +36,15 @@ Since oVirt 3.5.0 (VDSM 4.16.0), VDSM does not enable the overly verbose libvirt
 
 ## Reading the log
 
+### Verbosity and debug information
+
+Vdsm produces a huge amount of logging, and to improve the readability and to ease the troubleshooting, effort are ongoing to make the output tunable, to show messages from only some selected subsystems. As part as this effort, Vdsm gained the ability to change the log levels at runtime, without requiring a restart of the daemon. For example, to enable all the debug logs for the `virt` subsystem:
+
```
Comment (sandrobonazzola) [inline source/develop/developer-guide/vdsm/log-files.html.md]: Maybe Vdsm -> VDSM

Comment (sandrobonazzola) [inline source/develop/developer-guide/vdsm/log-files.html.md]: also please fill the pull request template, it's missing signature accepting contribution guidelines

**Comment by sandrobonazzola** [comment]
@mojaves any update?

File: source/develop/developer-guide/vdsm/log-files.html.md [hunk 3]
```diff
@@ -36,6 +36,15 @@ Since oVirt 3.5.0 (VDSM 4.16.0), VDSM does not enable the overly verbose libvirt
...
+      # vdsClient -s 0 setLogLevel DEBUG virt
```
Comment (michalskrivanek) [inline source/develop/developer-guide/vdsm/log-files.html.md]: @mojaves, well, this took so long that vdsClient is no longer relevant:/ can you please update it quickly?

**Comment by mojaves** [comment]
@fromanirh (I also switched github user meantime :))

**Comment by sandrobonazzola** [comment]
@fromanirh any update?

**Comment by fromanirh** [comment]
updated after too long, required a forced push on my repo, but hopefully will apply cleanly to ovirt-site

**Pull request merged**